connect/models/asset_request.py

```diff
@@ -131,7 +133,10 @@ def get_conversation(self, config=None):
         from connect.resources.base import ApiClient
 
         client = ApiClient(config, base_path='conversations')
-        response, _ = client.get(params={'instance_id': self.id})
+        try:
+            response, _ = client.get(params={'instance_id': self.id})
```

Comment:
I think we need to catch here more specific error, like return None only in case of 404. As I remember that is the answer, when there is no any conversation here